lib/rokt_sdk.dart

```diff
@@ -85,13 +85,15 @@ class RoktSdk {
   /// - Parameters:
   ///   - viewName: The name that should be displayed in the widget
   ///   - attributes: A string map containing the parameters that should be displayed in the widget
+  ///   - roktConfig: Rokt SDK configuration object
   ///   - onLoad: Function to execute right after the widget is successfully loaded and displayed
   ///   - onUnLoad: Function to execute right after widget is unloaded, there is no widget or there is an exception
   ///   - onShouldShowLoadingIndicator: Function to execute when the loading indicator should be shown
   ///   - onShouldHideLoadingIndicator: Function to execute when the loading indicator should be hidden
   static Future<void> execute(
       {required String viewName,
       Map<String, String> attributes = const {},
+      RoktConfig? roktConfig,
```

Comment:
Can we update the `README.md` as well?

Reply:
https://github.com/ROKT/rokt-sdk-flutter/pull/98